core/test/test_serial.cpp

```diff
@@ -76,3 +69,18 @@ TEST_F(ConnectConnect, FailSucc) {
 
 	EXPECT_FALSE(t.plan());
 }
+
+// https://github.com/ros-planning/moveit_task_constructor/issues/485#issuecomment-1760606116
+// TODO: Solutions are enumerated multiple times
```

Comment:
Did you fix this with `oit_to_enable` further above now or is this a separate issue?

Reply:
Sorry, this TODO is obsolete. It was resolved by oit_to_enable as you guessed.